src/main/java/it/gov/pagopa/reward/notification/dto/mapper/RewardsNotificationMapper.java

```diff
@@ -21,13 +21,14 @@ public RewardsNotification apply(String notificationId, LocalDate notificationDa
                 .initiativeName(rule.getInitiativeName())
                 .organizationId(rule.getOrganizationId())
                 .organizationFiscalCode(rule.getOrganizationFiscalCode())
-                .userId(trx.getUserId())
+                // TODO handle missing merchantId
```

Comment:
after the creation of the result, before to return it, throw IllegalArgumentException describing the assence of the beneficiaryId